### ISSUE TITLE
Download Play LLE doesn't work for Mario Party: Island Tour

### ISSUE THREAD

**Issue by AnotherUser97** [citra-emu/citra#5642]
I am trying to play Mario Party: Island Tour with my cousin using Download Play LLE. To be able to connect to the same room in Citra, we use a VLAN software (Hamachi). I have been able to get Local Multiplayer to work for games like Super Smash Bros. for Nintendo 3DS and Mario Party: The Top 100. However, when I tried to use Download Play LLE it does not work: for me my cousin's username will keep flashing and then disappear while for him "No software found" will appear on the Download Play app. I have dumped System Archives, the "titles" folder from my NAND, my config savegame, enabled DLP LLE via "View > Debugging > Toggle LLE Modules" (and so has my cousin). The issue in hand can be seen in [this video](https://drive.google.com/file/d/1KBu2ZqILqtceF1kakpIrVQZhCHPjcznb/view). The same issue occurs on Mario Party: Star Rush and Mario Kart 7, although in Mario Kart 7, if I press Start when his name is flashing, it goes to a loading menu and gets stuck (while my cousin's end still says No software found). For the other 2 games, pressing end while his name flashes does absolutely nothing.

I am using Citra built from source a day ago on Ubuntu 20.04.1 LTS while my cousin is using the latest Nightly build on Windows 10

**Comment by MojoJojoDojo** [comment]
I never could get DLP LLE to work over the internet.
It's too unstable for it even if the ping you have is 30-40 ms.
The only way to get it to work is to play locally on 1 PC with 2 or more Citra instances and use a screensharing program like Parsec.

**Comment by AnotherUser97** [comment]
Oh so it does have latency requirement?

**Comment by MojoJojoDojo** [comment]
I guess there are if I could'nt get it to work over the internet but it worked locally.
I tried with someone from the same country even.

**Comment by AnotherUser97** [comment]
Well I hope the citra team figures something out to allow more stable connections. I mean my ping is like 20-30ms, and you're saying that's not enough lol.

**Comment by B3n30** [comment]
When we tested Download Play LLE I was able to connect to another dev over the internet. But this dev lived close to me and we both had a very good internet connection (ping < 10ms). 

Remember that Nintendo designed Download Play to only work in wifi and not over the internet. But in a distant future when we have DownloadPlay HLE it might be possible to use Download Play with bad (normal) internet connection

**Comment by AnotherUser97** [comment]
I did think the same too. I remember the og DS download play also having ridiculously strict latency and timing because they never intended it to be work over the internet. I do hope an HLE DLP comes out in the future. PS: are there any plans of working on DLP HLE anytime?

**Comment by B3n30** [comment]
ATM most Citra devs are a bit in an hiatus so I wouldn't expect it anytime soon, but Citra is open source so one can never know

**Comment by AnotherUser97** [comment]
Understandable. Does the multiple instances method work? If so I need help setting up 2 citra instances on Linux.

**Comment by B3n30** [comment]
Yes 2 Citra instances on one PC should work without issues. For help about this please go to our discord channel. This isn't a support forum

**Comment by AnotherUser97** [comment]
Oh sorry about that. Thanks for letting me know.
